Runtime/Data/Attribute/RestrictedAttribute.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace Zinnia.Data.Attribute
+{
+    using UnityEngine;
+    using System;
+
+    /// <summary>
+    /// Draws the member in a specified restricted way.
```

Comment:
```suggestion
    /// Draws the property in a specified restricted way.
```
You used that in the docs you added and I think that's what Unity uses, too.